tools/flaky-test-reporter/config.go

```diff
@@ -34,10 +34,28 @@ const (
 )
 
 var (
-	jobConfigs = []JobConfig{
-		{"ci-knative-serving-continuous", "serving", prow.PostsubmitJob}, // CI flow for serving repo
+	// jobConfigs lists all repos and jobs to run within those repos
+	jobConfigs = map[string][]JobConfig{
+		// CI flows for serving repo
+		"serving": {{Name: "ci-knative-serving-continuous", Type: prow.PostsubmitJob, PostIssue: true},
+			{Name: "ci-knative-serving-istio-1.0.7-mesh", Type: prow.PostsubmitJob, PostIssue: true},
+			{Name: "ci-knative-serving-istio-1.0.7-no-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+			{Name: "ci-knative-serving-istio-1.1.2-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+			{Name: "ci-knative-serving-istio-1.1.2-no-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+		},
 	}
-	repoIssueMap = map[string]string{
+	// slackChannelsMap lists which slack channel to post results in for each job in repo
+	slackChannelsMap = map[string]map[string][]slackChannel{
+		// channel mapping for serving repo
+		"serving": {"default": {{"api", "CA4DNJ9A4"}},
```

Comment:
I dont think we should have a default(catch-all) here. If we dont define a repo, we will not send the notification. We should not send it to a random one.

Reply:
I am open to ideas. @chaodaiG implied he would like a fallback option, and I was assuming serving-api was the "standard" channel for the serving test jobs. I will modify this to whichever is desired.

Reply:
@srinivashegde86 's suggestion makes sense to me, we can remove the fall back logic

Reply:
Just to be clear, the initial thought of this fall back logic is for `getSlackChannel` method being able to get a channel just by repo name, since this repo+jobname pattern doesn't seem to apply in other repos